README.md

```diff
@@ -1,28 +1,65 @@
 # OmbuLabs::Auth
-Short description and motivation.
+> _Refactor a rack middleware (probably a ruby gem) out of https://points.ombulabs.com/ so that we can easily add GitHub-based authentication to other Ruby/Rails internal applications._
+>
+> _It would be useful if we could add it to http://skunk.fastruby.io (backend for skunk) so that anybody who is part of our organization could sign in and review details in the admin panel._
+>
+> _Said gem should be easily configurable (only with environment variables) and should make it very simple to add "OmbuLabs-authentication" to our internal Ruby apps._
```

Comment:
This is a little too verbose and doesn't concisely explain what the gem does... 

What if you shortened it like this? 

> This gem provides an easy way to generate new (Devise) sessions for members of a GitHub organization. 
> 
> If a user is signing in with GitHub and they are a (public) member of the configured GitHub organization, they will be allowed in.

Reply:
My apologies! Since this is meant to be used as an internal gem, I didn't give it much thought... 😜